dashboard/db/migrate/20161021202609_add_index_to_levels_on_name.rb

```diff
@@ -0,0 +1,7 @@
+class AddIndexToLevelsOnName < ActiveRecord::Migration[5.0]
+  def change
+    change_table(:levels) do |t|
+      t.index :name
+    end
```

Comment:
I would think that add_index, being more specific, would be preferred? No strong preference, though, if they ultimately make the same SQL call under the hood.

Apart from that, LGTM!